lib/cli.js

```diff
@@ -37,7 +34,7 @@ require('yargs')
           alias: 't',
           default: 'vue',
           desc: chalk.gray(y18n.__('init.theme')),
-          choices: ['vue', 'buble', 'dark', 'pure'],
+          choices: ['vue'],
           nargs: 1,
```

Comment:
There is no choice for theme now，is it necessary to still hold it?

Reply:
Let's keep it